stingray/lightcurve.py

```diff
@@ -27,8 +27,11 @@ class Lightcurve(object):
 
     Parameters
     ----------
-    time: iterable
-        A list or array of time stamps for a light curve
+    time: Iterable
```

Comment:
I find this docstring a bit confusing as it is written now. I'd be explicit that it can take an `astropy.Time` object:
Maybe this docstring should read:

```
time: iterable, `:class:astropy.time.Time` object or `:class:astropy.units.Quantity` object
```

Reply:
Done